config/routes.rb

```diff
@@ -8,4 +8,5 @@
   get '/reservations', to: 'reservations#index'
   get '/create_hall', to: 'halls#new'
   get '/ticket_desks', to: 'ticketdesks#index'
+  get '/create_reservation', to: 'reservations#new'
```

Comment:
In future - consider to configure routes in bit different way - [link to guide](https://guides.rubyonrails.org/routing.html#resource-routing-the-rails-default)

For instance:
```
# bad
get '/halls', to: 'halls#index' # => /halls
get '/create_hall', to: 'halls#new' # => /create_halls

# good
resources :halls, only: [:index, :create]
# => GET /halls
# => POST /halls
```

Article to read more about it - https://www.bigbinary.com/learn-rubyonrails-book/restful-routes-in-ruby-on-rails